src/triggers/Trigger_Account.trigger

```diff
@@ -0,0 +1,17 @@
+/*
+  *  Purpose         :    Adding prefix to account number and sending mails
+  *  Created By      :    Leena Ahuja
+  *  Created Date    :    2022/07/20
+  *  Revision Logs   :    V_1.0 - Created - 2022/07/19
+*/
+trigger Trigger_Account on Account (before insert, after insert) {
+    if(Trigger.isInsert && Trigger.isAfter){
+        AccountTriggerHelper.afterInsertMail(Trigger.New);       
+    }
+    if(Trigger.isInsert && Trigger.isBefore){
+        AccountTriggerHelper.addAccountPrefix(Trigger.New);
+    }
+    if(Trigger.isUpdate){
```

Comment:
This will never invoke as before update or after update is not included in trigger parameters.